app/renderer/components/tabs/content/audioTabIcon.js

```diff
@@ -56,6 +58,39 @@ class AudioTabIcon extends React.Component {
     return props
   }
 
+  componentDidMount (props) {
+    this.transitionIfRequired()
+  }
+
+  componentDidUpdate (prevProps) {
+    this.transitionIfRequired(prevProps)
+  }
+
+  transitionIfRequired (prevProps) {
+    // transition only if we have an element and we're not pinned
+    if (!this.element || this.props.isPinned) {
+      return
+    }
+    // transition only if audio state changes
+    const hasChanged = !prevProps || this.props.showAudioIcon !== prevProps.showAudioIcon
+    // TODO: if we want to animate the unmounting of the component (when audio is stopped),
+    //       then we should use https://github.com/reactjs/react-transition-group
```

Comment:
nit: just taking advantage of a needed rebase, could you normalize the comment here? like
```diff
-//       TODO:
+// TODO:
```

Reply:
@cezaraugusto just to confirm - by normalize do you mean you don't want the text with the same indentation as I have it? You want it to wrap underneath the 'TODO:' from the previous line?